app/views/profiles/_form.html.slim

```diff
@@ -9,16 +9,17 @@
       = f.input :first_name
       = f.input :last_name
       = f.input :phone
-      = f.input :avatar, as: :file
+      = f.input :avatar, label: _('Avatar'), as: :file
       = f.input :address
       = f.input :profession
 
   .row
     .col-xs-12
-      h2 Arbeitstage
+      h2 =_('Working days')
       ul.list-inline
         - [:monday, :tuesday, :wednesday, :thursday, :friday].each do |day|
           li= f.input day, as: :boolean
   .row
     .col-xs-12
-      = f.button :submit, class: 'btn btn-default'
+      = f.button :submit, ( f.object.new_record? ? _('Create Profile') : _('Update Profile')),
```

Comment:
Because the form is rendered in two views, each one has a different button and I needed the text of each to be translated.